diff --git a/src/jsonrpccpp/CMakeLists.txt b/src/jsonrpccpp/CMakeLists.txt
index 2d047d7..73e4462 100644
--- a/src/jsonrpccpp/CMakeLists.txt
+++ b/src/jsonrpccpp/CMakeLists.txt
@@ -237,7 +237,6 @@ if (WIN32 AND NOT HUNTER_ENABLED)
     file(GLOB win32_libs ${CMAKE_SOURCE_DIR}/win32-deps/lib/*.dll)
     file(COPY ${win32_libs} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
     install(FILES ${win32_libs} DESTINATION bin)
-    install(DIRECTORY ${CMAKE_SOURCE_DIR}/win32-deps/include DESTINATION .)
 endif ()
 
 install(TARGETS ${ALL_LIBS} EXPORT libjson-rpc-cppTargets
diff --git a/src/jsonrpccpp/client/connectors/windowstcpsocketclient.cpp b/src/jsonrpccpp/client/connectors/windowstcpsocketclient.cpp
index 868668b..156eabd 100644
--- a/src/jsonrpccpp/client/connectors/windowstcpsocketclient.cpp
+++ b/src/jsonrpccpp/client/connectors/windowstcpsocketclient.cpp
@@ -27,7 +27,7 @@ WindowsTcpSocketClient::WindowsTcpSocketClient(const std::string &hostToConnect,
 
 WindowsTcpSocketClient::~WindowsTcpSocketClient() {}
 
-void WindowsTcpSocketClient::SendRPCMessage(const std::string &message, std::string &result) throw(JsonRpcException) {
+void WindowsTcpSocketClient::SendRPCMessage(const std::string &message, std::string &result) {
   SOCKET socket_fd = this->Connect();
   char buffer[BUFFER_SIZE];
   bool fullyWritten = false;
@@ -117,7 +117,7 @@ string WindowsTcpSocketClient::GetErrorMessage(const int &e) {
   return message;
 }
 
-SOCKET WindowsTcpSocketClient::Connect() throw(JsonRpcException) {
+SOCKET WindowsTcpSocketClient::Connect() {
   if (this->IsIpv4Address(this->hostToConnect)) {
     return this->Connect(this->hostToConnect, this->port);
   } else // We were given a hostname
@@ -162,7 +162,7 @@ SOCKET WindowsTcpSocketClient::Connect() throw(JsonRpcException) {
 }
 
 SOCKET
-WindowsTcpSocketClient::Connect(const string &ip, const int &port) throw(JsonRpcException) {
+WindowsTcpSocketClient::Connect(const string &ip, const int &port) {
   SOCKADDR_IN address;
   SOCKET socket_fd = socket(AF_INET, SOCK_STREAM, 0);
   if (socket_fd == INVALID_SOCKET) {
